css-view-transitions-1/Overview.bs

```diff
@@ -1424,19 +1437,20 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 		1. [=Assert=]: |transition|'s [=ViewTransition/phase=] is [=phases/before=] "`update-callback-called`".
 
-		1. Let |callbackPromise| be [=a new promise=] in |transition|'s [=relevant Realm=].
+		1. Let |callbackPromise| be null.
 
-			* If |transition|'s [=ViewTransition/update callback=] is null, then resolve |callbackPromise|.
+		1. If |transition|'s [=ViewTransition/update callback=] is null,
+			then set |callbackPromise| to [=a promise resolved with=] undefined,
+			in |transition|'s [=relevant Realm=].
 
-			* Otherwise, let |callbackPromise| be the result of [=/invoking=] |transition|'s [=ViewTransition/update callback=].
+		1. Otherwise, set |callbackPromise| to the result of [=/invoking=] |transition|'s [=ViewTransition/update callback=].
 
 		1. Set |transition|'s [=ViewTransition/phase=] to "`update-callback-called`".
 
-		1. [=promise/React=] to |callbackPromise|:
-
-			* If |callbackPromise| was resolved, then [=resolve=] |transition|'s [=ViewTransition/update callback done promise=].
+		1. [=Resolve=] |transition|'s [=ViewTransition/update callback done promise=]
+			with the result of [=reacting=] to |callbackPromise|:
 
-			* If |callbackPromise| was rejected with reason |r|, then [=reject=] |transition|'s [=ViewTransition/update callback done promise=] with |r|.
+			- If the promise was fulfilled, then return undefined.
```

Comment:
This is technically a normative change, but it doesn't change any currently observable behaviour. It protects us from possible race conditions in future.